hassio-to-pubsub-source/README.md

```diff
@@ -0,0 +1,41 @@
+# Publishing Home Assistant events to Pub/Sub
+
+## TODO
+
+- [ ] Home Assistant add-on
+- [ ] Use GCP service account for authentication (can be done with `GOOGLE_APPLICATION_CREDENTIALS`)
+- [ ] Implement filtering for certain events/domains/entities
+
+## You will need
+
+- Home Assistant running and accessible on the network
+- Access to the `a2i2-ua-smart-home` project on GCP (or a service account)
+- A [long-lived access token](https://www.atomicha.com/home-assistant-how-to-generate-long-lived-access-token-part-1/) from Home Assistant
+
+## Set config
+
+Open up `hass_to_pubsub.py` and set values as appropriate near the top of the file. You will need to set:
+
+- `HOME_ID` to something that identifies your house (e.g. `rohan-home`)
+- `ACCESS_TOKEN` to your long-lived access token
```

Comment:
Don't need to do this anymore, as we can set these variables via addon config (or environment variables if running locally). `ACCESS_TOKEN` is only required if running locally - it's obtained automatically if running via Supervisor.